app/views/admin/questions/edit.html.haml

```diff
@@ -0,0 +1,31 @@
+%h2.text-center Modifier Question
```

Comment:
Indeed, this view looks like a lot like the `new` one. We would probably be able to factorise them with partials later.
Could you please drop a `TODO` at the top of both views?